.github/ISSUE_TEMPLATE/feature-request.yml

```
@@ -0,0 +1,52 @@
+# Ultralytics 🚀 AGPL-3.0 License - https://ultralytics.com/license
+
+name: 🚀 Feature Request
+description: "Suggest an Ultralytics Handbook improvement"
+labels: [enhancement]
+type: "feature"
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for submitting an Ultralytics Handbook 🚀 Feature Request!
+
+  - type: checkboxes
+    attributes:
+      label: Search before asking
+      description: >
+        Please search the Ultralytics Handbook [site](https://handbook.ultralytics.com/) and [issues](https://github.com/ultralytics/handbook/issues) to see if a similar feature request already exists.
+      options:
+        - label: >
+            I have searched https://github.com/ultralytics/handbook/issues and did not find a similar request.
+          required: true
+
+  - type: textarea
+    attributes:
+      label: Description
+      description: Briefly describe the feature you would like to see added to Ultralytics Handbook.
+      placeholder: |
+        What new capability or improvement are you proposing?
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Use case
+      description: Explain how this feature would be used and who benefits from it. Screenshots or mockups are welcome.
+      placeholder: |
+        How would this feature improve your workflow?
+
+  - type: textarea
```

💭 **SUGGESTION**: Similar to the bug template, the `Additional` field has no explicit `validations`. It’s optional by default, but adding a `validations` block (even with `required: false`) can improve consistency across templates and reduce ambiguity for future edits.

**Suggested change:**
```suggestion
  - type: textarea
```